### PR TITLE
docs: remember Windows users to install Git

### PR DESCRIPTION
**Pull request by cironunes**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
The docs don't tell Windows users to install Git which is a prerequisite of the Angular CLI

## What is the new behavior?
Docs updated to tell Windows users to install Git.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/quickstart.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ This Quick Start uses the [npm client](https://docs.npmjs.com/cli/install) comma
 To check that you have the npm client installed, run `npm -v` in a terminal/console window.
 
 
+{@ windowsusers}
```
Comment (brandonroberts) [inline aio/content/guide/quickstart.md]: This should be a more general section like `### Git`

```
Some description of Git with a link to installation

To check that you have the git client installed, run `git --version` in a terminal/console window.
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: aio/content/guide/quickstart.md (hunk 1 above)
Comment (cironunes) [reply aio/content/guide/quickstart.md]: cool. I'll update the PR

**Comment by brandonroberts** [comment]
@cironunes are you still interested in making these changes?

**Comment by brandonroberts** [comment]
Reopen this PR or open a new one when you're ready. Thanks

**Pull request closed without merging**